outlookmail/tests/test.bal

```diff
@@ -428,44 +457,28 @@ function testCreateMailSearchFolder() {
     }
 }
 
-@test:Config {
-    enable: true,
-    dependsOn: [testListAttachment]
-}
-function testAddLargeFileAttachment() returns @tainted error? {
-    log:printInfo("outlookClient->TestAddLargeFileAttachments");
-    stream<io:Block, io:Error?> blockStream = check 
-    io:fileReadBlocksAsStream("outlookmail/tests/sample.pdf", 3000000);
-    var output = outlookClient->addLargeFileAttachments(sentMessageId, "myFile.pdf", blockStream, fileSize = 10635049);
-    if (output is error) {
-        test:assertFail(msg = output.toString());
-    }
-}
-
-@test:Config {
-    enable: true,
-    dependsOn: [testAddLargeFileAttachment]
-}
-function testDeleteAttachment() returns @tainted error? {
-    log:printInfo("outlookClient->testDeleteAttachment()");
-    var output = outlookClient->deleteAttachment(sentMessageId, attachmentId);
-    if (output is error) {
-        test:assertFail(msg = output.toString());
-    }
-}
 
 @test:AfterSuite  {}
 function afterFunc() returns error? {
     log:printInfo("Removing sent messages");
-    var output = outlookClient->listMessages(folderId = "sentitems", 
-    optionalUriParameters = "?$select: \"sender,subject,hasAttachments\"&top=2");
-    if (output is stream<Message, error?>) {
-        _ = check output.forEach(function(Message queryResult) {
-            string messageID = queryResult?.id.toString(); 
-            http:Response|error result =  outlookClient->deleteMessage(messageID, "sentitems");
-            if (result is error) {
-                test:assertFail(msg = result.toString());
-            }
-        });
+    int index = 0;
+    while true {
+        var output = outlookClient->listMessages(folderId = "sentitems", 
+            optionalUriParameters = "?$select: \"sender,subject,hasAttachments\"&top=2");
+        if (output is stream<Message, error?>) {
+            _ = check output.forEach(function(Message queryResult) {
+                string messageID = queryResult?.id.toString(); 
+                http:Response|error result =  outlookClient->deleteMessage(messageID, "sentitems");
+                if (result is error) {
+                    test:assertFail(msg = result.toString());
+                } else {
+                    index = index + 1;
```

Comment:
Do we need to use a counter for this purpose? can't use a boolean flag instead? that would be more clearer IMO